From: Andrew Morton <akpm@linux-foundation.org>
Subject: scripts/: make .sh files executable
Date: Sun Feb 26 12:50:54 PM PST 2023

scripts/spdxcheck-test.sh and scripts/test_fortify.sh don't have their x
bits set.

I really only did this to check whether `git diff --no-index --' can be
used to replace `diff', so as to preserve file permissions.

Suggested-by: Linus Torvalds <torvalds@linux-foundation.org>
  Link: https://lkml.kernel.org/r/CAHk-=wj-+PTUfu9NsdDt3p7tLXbUH-KJPBz+r4wHX075fydEOQ@mail.gmail.com
Cc: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Paul Eggert <eggert@cs.ucla.edu>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


diff --git a/scripts/spdxcheck-test.sh~scripts-make-sh-files-executable b/scripts/spdxcheck-test.sh
old mode 100644
new mode 100755
diff --git a/scripts/test_fortify.sh~scripts-make-sh-files-executable b/scripts/test_fortify.sh
old mode 100644
new mode 100755
_
